railties/lib/rails/generators/rails/app/templates/config/initializers/content_security_policy.rb.tt

```diff
@@ -18,7 +18,7 @@
 #
 #   # Generate session nonces for permitted importmap and inline scripts.
```

Comment:
I think this comment needs to be updated as well.
```suggestion
#   # Generate session nonces for permitted importmap, inline scripts and inline styles.
```

Reply:
Thank you for the review 🙇  . I applied your suggestion.